chapters/batching.rst

```diff
@@ -0,0 +1,131 @@
+=============================================
+ Understanding Batching in Jina
+=============================================
+
+Some :term:`Executors` in Jina may profit from the capacity to process multiple instances of their input at the same time.
+This can lead to a speed up and higher throughput.
+
+In order to provide this capability, the :term:`Driver` extracts all the needed information from all the :term:`Document` in a :term:`Request`. [see `guide on request size`_].
```

Comment:
Should add a link for `guide on request size`_ maybe? I can add this if needed.

Reply:
Can u do so? I thought the link was added

Reply:
Yes I added.